shell.php

```diff
@@ -32,6 +32,46 @@ function featureShell($cmd, $cwd) {
     );
 }
 
+function executeCommand($cmd) {
```

Comment:
Please move this function at the top (under `expandPath()`) to keep all `feature*()` function grouped :)